src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.page {
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-width;
+    padding: 0;
+    margin: auto;
+    gap: $gap;
```

Comment:
Ensure that the variable `$gap` is defined. If it's not defined in this file, it should be imported from another SCSS file.